.vscode/tasks.json

```diff
@@ -198,23 +235,25 @@
       "problemMatcher": "$msCompile"
     },
     {
-      "label": "Web App: Dotnet Publish (self-contained) - net6.0",
+      "label": "tasks: dotnet publish release (self-contained) - net6.0",
       "dependsOn": [
-        "Cleanup build folder",
-        "Dotnet Publish (self-contained) - net6.0"
+        "cleanup publish folder",
+        "dotnet publish (self-contained) - net6.0"
       ]
     },
     {
-      "label": "Web App: Dotnet Publish (framework dependent) - net6.0",
+      "label": "tasks: dotnet publish release (framework dependent) - net6.0",
       "dependsOn": [
-        "Cleanup build folder",
-        "Dotnet Publish (framework dependent) - net6.0"
+        "cleanup publish folder",
+        "dotnet publish (framework dependent) - net6.0"
       ]
     },
     {
-      "label": "Gulp Build",
+      "label": "gulp build",
+      "type": "shell",
+      "command": "gulp",
       "args": [
-        "gulp build"
```

Comment:
Should this be npm run build instead ?

Reply:
never tried but that's for debugging the gulpfile.js

Reply:
Yes, but this required gulp to be installed globally. `npm run build` does not as it uses gulp from node_modules.

Reply:
That's the default task suggested. npm run build will try to install it if not found every time whereas this will just execute what is needed knowing that it is already installed globally or locally. I don't mind changing it, but need to test it first.

Reply:
Npm run build does not install first as far as I know. It simply runs a script as defined in the package.json script section.

Reply:
You're right. But then it will do the same if the gulp package is not installed. So, here it is just a task that executes the gulp build command to debug it without knowing if it's installed or not. What you suggest is to run the gulp pipeline by passing through npm which could lead to executing also something else than gulp.

Reply:
To be honest, we keep it there because it was there and I updated the task to comply with the latest VS Code tasks implementation details; but it's not something that we use much. Also, it is looking like we might need to move to WebPack or something else at some point.